ui/packages/catalog-ui-search/src/main/webapp/component/result-selector/result-selector.view.js

```diff
@@ -146,6 +149,35 @@ var ResultSelector = Marionette.LayoutView.extend({
       this.onBeforeShow
     )
   },
+  startListeningToSelectedResults: function() {
```

Comment:
Instead of updating this view to know about the selection checkbox internals (seen in `handleSelectionChange`), I think we should pass the selection interface into the checkbox selection checkbox component and encapsulate the logic there.

We might have to make two different components if we go that route.  One for the individual items, one for the select all / deselect all.